src/google-apis/auth-client.ts

```diff
@@ -18,7 +18,8 @@
 import * as common from '@google-cloud/common';
 const pkg = require('../../../package.json');
 import * as is from 'is';
-var isFunction = is.fn;
+// TODO: Address the error where `is` does not have a `fn` property
+var isFunction = (is as {} as {fn: Function}).fn;
```

Comment:
Use `is.function`: `is.fn === is.function`

Reply:
Awesome.  I'll update it.

Reply:
I think I'll leave this for now using `is.fn`.  In a follow-up PR we can update the code to use `is.function`.